tap_appstore/__init__.py

```diff
@@ -221,20 +227,28 @@ def query_report(api: Api, catalog_entry):
             rep = _attempt_download_report(api, report_filters)
 
             # write records
```

Comment:
please add a comment why we are doing a try/except. Sometimes the report for today throws an error.